builtin/credential/appgroup/path_tidy_user_id.go

```diff
@@ -0,0 +1,77 @@
+package appgroup
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathTidyUserID(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "tidy/user-id$",
+
```

Comment:
Probably a good idea to have a safety buffer to account for clock skew and other issues, since we're checking validity at login time anyways.